cli/context/docker/load.go

```diff
@@ -127,6 +127,8 @@ func (c *Endpoint) ClientOpts() ([]client.Opt, error) {
 	version := os.Getenv("DOCKER_API_VERSION")
 	if version != "" {
 		result = append(result, client.WithVersion(version))
```

Comment:
WithVersion should now ignore empty values, so just adding both WithVersion and WithAPIVersionNegotiation should also work